styles.css

```diff
@@ -69,3 +110,135 @@
 .contact-info-button {
   margin-right: 30px;
 }
+
+/* Recent works */
+.recent-works {
+  margin-bottom: 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.works-cards-container {
+  width: 90%;
+  margin-top: 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.work-card {
+  max-width: 369px;
+  margin-left: 5%;
+  margin-right: 5px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+  border: 1px solid #d0d9d4;
+  margin-bottom: 15px;
+  align-self: center;
+}
+
+.project-image {
+  height: 252px;
+  background-color: #ebf0ee;
+}
+
+.recent-works-frameworks {
+  display: flex;
+  justify-content: space-evenly;
+  margin-bottom: 40px;
+}
+
+.frameworks-item {
+  padding: 12px 12px;
+  font-family: 'Inter', sans-serif;
+  background: #ebf0ee;
+  margin-right: 5px;
+}
+
+.btn-align {
+  margin-bottom: 10px;
+}
+
+/* About me */
+.about-me {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+}
+
+.about-me p {
+  font-family: 'Inter', sans-serif;
+  font-size: 20px;
+  margin-bottom: 30px;
+}
+
+.about-me-headline {
+  width: 100%;
+  height: 350px;
+  background-image: url("images/illustration-1-about-me-mobile.jpg"), url("images/illustration-2-about-me-mobile.jpg");
+  background-repeat: no-repeat;
+  background-position: top right, bottom left;
+}
+
+.knowledges {
+  display: grid;
+  align-content: center;
```

Comment:
- [ ] Kindly decrease the size of the skills cards because it is not fitting on phone screens. Keep shining 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/158912114-e1b1325c-6bf3-4079-a83b-6bc1f992bcdf.png)